packages/runtime-common/router.ts

```diff
@@ -49,7 +49,7 @@ export class Router {
     routes.set(path, handler);
   }
 
-  async handle(request: Request): Promise<Response> {
+  async handle(request: Request, response: Response): Promise<Response> {
```

Comment:
i don't believe we would pass in the response here. rather this is where the response is created (since this is the first step in the request handling logic).